backend/server/routers/courses.py

```diff
@@ -25,6 +25,8 @@
     tags=["courses"],
 )
 
+require_uid = HTTPBearerToUserID()
+
 # TODO: would prefer to initialise ALL_COURSES here but that fails on CI for some reason
 ALL_COURSES: Optional[Dict[str, str]] = None
```

Comment:
Since we are finally getting a proper startup with redis, we should look into getting this fixed

Reply:
real

Reply:
(will do in future)